### DISCUSSION TITLE
Specify work directory to run simulations in

### DISCUSSION THREAD

**Discussion by mvanriet** [nunobrum/PyLTSpice#70] [Ideas]
When running a lot of different simulations, you get a lot of output files.
It may be convenient to have these output files in a separate folder.

**Comment by nunobrum** [comment]
It's a good idea. How would this be implemented ?
I can think about three possibilities, not mutually exclusive:
a) As a optional parameter in the SimCommander constructor
b) As a class method of the SimCommander constructor
c) As a optional parameter on the run() method.

Is there any other option that I'm missing? 
A few personal considerations which I think make sense.
As an optional parameter in the constructor, if it is not given , the same directory of the .net or .asc file is used.
As an optional parameter of the run() method, if it is not given, it uses the last directory used.

**Comment by nunobrum** [comment]
An working implementation of this feature is now on 'dev_space' branch.
Finally implemented as an optional parameter of the SimCommander constructor.
